docs/setup/k8s/running-in-production-k8s.md

```diff
@@ -0,0 +1,108 @@
+# Running Nuclio over Kubernetes in production
+
+After familiarizing yourself with Nuclio, and [deploying it over k8s](/docs/setup/k8s/getting-started-k8s.md) you may find yourself looking for extra configuration knobs, and proper practices for using it in a production environment.
```

Comment:
Not sure I wouldn't obsolete the resources altogether and describe installation only with helm. We don't update them and they may already be broken

Reply:
Since getting started docs currently rely on them, I think we should do this a bit later on, once we have a nice getting started with helm, and possibly tested some scenarios with helm3